query/src/users/auth/auth_mgr.rs

```diff
@@ -56,11 +56,15 @@ impl AuthMgr {
     pub async fn auth(&self, credential: &Credential) -> Result<UserInfo> {
         match credential {
             Credential::Jwt { token: t } => {
-                let user_name = match &self.jwt {
+                let (user_name, is_supper) = match &self.jwt {
                     Some(j) => j.get_user(t.as_str())?,
                     None => return Err(ErrorCode::AuthenticateFailure("jwt auth not configured.")),
                 };
-                self.users.get_user(&self.tenant, &user_name, "%").await
+                if is_supper {
+                    Ok(UserInfo::new_supper(user_name))
```

Comment:
maybe the super user's granted privileges would be empty in this case 🤔

some unit tests may help us making the semantics of superuser more clear

Reply:
> maybe the super user's granted privileges would be empty in this case 🤔
it was default(), I just changed to empty()

Reply:
maybe the superuser has more privileges than root?  q.q

if it's root user, it'd get all the privileges

Reply:
I though superuser do not use privileges at all.